### ISSUE TITLE
code coverage user interface is pretty different now...

### ISSUE THREAD

**Issue by jimandreas** [googlecodelabs/android-testing#46]
In the step "Bonus: Code Coverage" the latest Android Studio version (2.1.1) is quite different from what is discussed in:

[https://codelabs.developers.google.com/codelabs/android-testing/#10](https://codelabs.developers.google.com/codelabs/android-testing/#10)

After some fumbling around I found the code coverage controls and am now happily experimenting with them.   There does seem to be some documentation here that matches what I see in Android Studio:

[https://www.jetbrains.com/help/idea/2016.1/viewing-code-coverage-results.html](https://www.jetbrains.com/help/idea/2016.1/viewing-code-coverage-results.html)

Basically a right click on the tests in the Project view provides a "Run soAndSo with Coverage" while mysteriously this is sometimes greyed out in the Run menubar drop-down.  The "Coverage" side tab or View then Tool Windows then Coverage selection gives you the Coverage view of the underlying code.   And the various packages are then annoted in the Project window with the percentage of test coverage.   Rather nice!!

![image](https://cloud.githubusercontent.com/assets/15666465/15364697/baecf3a6-1d1d-11e6-9740-8f1ce69311ee.png)

**Comment by jfschmakeit** [comment]
The steps in our codelab aren't too different. We are setting up a test suite for the project/target first, then run the coverage report on it.

I found that the option is grayed out in the menu if you haven't selected the correct run target or the appropriate package. The UI is a bit confusing here unfortunately.

I have updated the screenshots in this particular code lab step and explained how you can reopen the coverage report again (Analyze > Coverage), but it looks to me like it's still uptodate. Our aim was here just to call out the feature and show how to get started with it, there are a few other options that we couldn't cover in this step.

Was there anything in particular that you felt was missing/unclear?

**Comment by jimandreas** [comment]
Here are a few suggestions for clarification.   In the step:

> On the next dialog, select the "Code Coverage" tab.

change to:

> In the Run... / Edit Configurations dialog, select the "Code Coverage" tab.

Next I hit the "Empty Test Suite" problem.   

    ---- IntelliJ IDEA coverage runner ---- 
    sampling ...
    include patterns:
    exclude patterns:
    Process finished with exit code 1
    Class not found: "com.example.android.testing.notes.data.InMemoryNotesRepositoryTest" Empty test suite.

See this issue:   https://github.com/googlecodelabs/android-testing/issues/27

But if I manually run each subfolder in notes(test) - it seems to work, and then I can run all tests without getting the "Empty test suite" error.

Now the step is:

> Next, run the code coverage report. In the build configuration, select the new build configuration we have just created, and click the code coverage button  to generate the report.

But the button in the Coverage pane looks like the one documented here:

https://www.jetbrains.com/help/idea/2016.3/coverage-tool-window.html

> 	Click this button to generate a code coverage report and save it to the specified directory. See Generating Code Coverage Report for details.

(a rectangle with an up arrow) https://www.jetbrains.com/help/img/idea/2016.3/exportToTextFile.png

Overall things seem to work well - as of Android Studio version 2.2.2.  Thank you for all the work setting up this code lab!!

**Comment by jfschmakeit** [comment]
Cheers, I updated the wording slightly, and updated the screenshots. This should go live in the next push. 
(BTW, the icon looks different for me again on Android Studio 2.2.3 - I think there could be a difference in the IntelliJ/Android Studio theme.)
